inflammation/models.py

```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
```

Comment:
This function actually returns a dicct and not the standard deviation

Reply:
well spotted, will change it

Reply:
good change!

Reply:
function adapted, see commit [914e1ac](https://github.com/elena-rae/python-intermediate-inflammation/commit/914e1ac4ccfc8cf4fbc7387172557a0697bfc7c1)